esmvaltool/install/R/r_requirements.yml

```diff
@@ -1,54 +0,0 @@
----
-# R packages to be installed
-# We support several installation procedures:
-
-# Latest version from CRAN:
-#  mypackage:
-
-# Specific version from CRAN (use only if last version can not be used):
-#   mypackage:
-#     origin: cran
-#     version: my_version
-
-# The next ones are NOT allowed on final code, but we support them to
-# facilitate developing and testing of recipes before merging if the CRAN
-# version is not available yet
-
-# From GitHub:
-#   mypackage:
-#     origin: github
-#     path: my_user/my_packge_repo@my_version
-
-# From a generic git:
-#   mypackage:
-#     origin: git
-#     url: my_repo_url
-#     ref: my_branch_commit_or_tag
-
-abind:
-akima:
-climdex.pcic:
-ClimProjDiags:
-docopt:
-dotCall64:
-functional:
-ggplot2:
-gridExtra:
-lintr:
-logging:
-mapproj:
-maps:
-multiApply:
-ncdf4:
-ncdf4.helpers:
-PCICt:
-plyr:
-RColorBrewer:
-Rcpp:
-s2dverification:
-snow:
-SPEI:
-styler:
-tools:
```

Comment:
I am not sure what is the tools package for, but in the other pull request I had to keep it in this file because it's not on corda-forge. Is it not needed afterall?

Reply:
Iirc, the R package `tools` is needed, but it is included in the conda-forge package `r-base`.

Reply:
Do you have more information on that @zklaus? Maybe a link to confirm that this is indeed the case?

Reply:
I hope you'll accept a stackoverflow answer: https://stackoverflow.com/a/9705725

Reply:
I'm still confused. Apparently, we were installing this package from CRAN but it is not even available on CRAN (as described in the StackOverflow post and absent from this list https://cran.r-project.org/web/packages/available_packages_by_name.html).

If it is this package, https://cran.r-project.org/bin/windows/Rtools/rtools42/rtools.html, we do not need to worry about it because we are no longer compiling from source and the documentation of this package says:
> Rtools42 is only needed for installation of R packages from source or building R from source.

Reply:
I think that is a different package (it is Rtools which would become r-rtools on conda-forge) but we still need not worry. That line in `r_dependencies.yml` likely never did anything because in `setup.R` we always filtered all `installed.packages()`, which contained `tools` because it's in `base`.

Reply:
> That line in r_dependencies.yml likely never did anything ..

Thanks for investigating, that was probably it then.

@sloosvel Does that answer your question too?

Reply:
I guess so